proposals/csharp-9.0/function-pointers.md

```diff
@@ -268,6 +268,92 @@ The better function member specification will be changed to include the followin
 
 This means that it is possible to overload on `void*` and a `delegate*` and still sensibly use the address-of operator.
 
+### Type Inference
+
+In unsafe code, the following changes are made to the type inference algorithms:
```

Comment:
>In unsafe code,  [](start = 0, length = 16)

Is "In unsafe code" necessary?

Reply:
Yes, all of the unsafe rules only apply in unsafe code.